source/math/FloatExtension.ooc

```diff
@@ -110,7 +110,7 @@ extend Float {
 				value /= 10.0f
 				radix *= 10.0f
 			}
-			while (Float absolute(value) < pow(10.0f, valueDigits-1)) {
+			while (Float absolute(value) - pow(10.0f, valueDigits-1) < -0.000001) {
```

Comment:
Right, epsilon is > 1E-7 for 32-bit `Float`. Thanks for the fix.

We should probably have a test for this...